source/grid/grid_out.cc

```diff
@@ -2461,13 +2464,68 @@ GridOut::write_svg(const Triangulation<2, 2> &tria, std::ostream &out) const
                                  (height -
                                   (height / 100.) * 2. * margin_in_percent))
                           << "\"/>" << '\n';
+
+
+                      if (svg_flags.label_boundary_id)
+                        {
+                          float distance_to_camera = std::sqrt(
```

Comment:
just make it a `double`. That's what `std::sqrt` returns, after all.

Also, I think you can make it `const`.

Reply:
I copied it from line 2293 here
https://github.com/dealii/dealii/blob/27f330ab0685777c0be4049263e33dc9d9f2e275/source/grid/grid_out.cc#L2293
but I can change both to `const double`s while I'm at it.

Reply:
Ah, thanks. Yes, that would be nice.